app/models.py

```diff
@@ -1,3 +1,246 @@
+            return None
+
+        current_datetime = datetime.datetime.now()
+        tweets = []
+
+        for diff in range(0, 6):
+            from_date = (current_datetime - datetime.timedelta(days=7-diff)).strftime('%Y-%m-%d')
+            to_date = (current_datetime - datetime.timedelta(days=6-diff)).strftime('%Y-%m-%d')
+
+            response = self.api.GetSearch(term='"'+movie.Title +'" -filter:links', since=from_date, until=to_date, lang='en', result_type='mixed')
+
+            for tweet in response:
+                # tag movie with imdbID
+                tweet.imdbID = movie.imdbID
+
+                # only append Tweets in English
+                if tweet.lang == 'en' or tweet.user.lang == 'en':
+                    tweets.append(tweet)
+
+        return tweets
+
+
+class Movie(models.Model):
+    Title = models.CharField(max_length=128)
+    Year = models.IntegerField(null=True, blank=True)
+    YomatoURL = models.CharField(max_length=1024, null=True, blank=True)
+    Actors = models.CharField(max_length=1024, null=True, blank=True)
+    BoxOffice = models.CharField(max_length=1024, null=True, blank=True)
+    Genres = models.CharField(max_length=1024, null=True, blank=True)
+    Director = models.CharField(max_length=1024, null=True, blank=True)
+    imdbRating = models.FloatField(null=True, blank=True)
+    tomatoRating = models.CharField(max_length=32, null=True, blank=True)
+    tomatorUserRating = models.CharField(max_length=32, null=True, blank=True)
+    plot = models.CharField(max_length=2048, null=True, blank=True)
+    tomatoConsensus = models.CharField(max_length=1024, null=True, blank=True)
+    Poster = models.CharField(max_length=1024, null=True, blank=True)
+    imdbID = models.CharField(max_length=1024)
+
+    param_defaults = {
+        'Title': None,
+        'Year': None,
+        'YomatoURL': None,
+        'Actors': None,
+        'BoxOffice': None,
+        'Genres': None,
+        'Director': None,
+        'imdbRating': None,
+        'tomatoRating': None,
+        'tomatoUserRating': None,
+        'Plot': None,
+        'tomatoConsensus': None,
+        'Poster': None,
+        'imdbID': None
+    }
+
+    def __unicode__(self):
+        return self.Title
+
+    def fillWithJsonObject(self, jsonObject):
+        """
+        :param jsonObject: a JSON Object containing information about a movie returned by the OMDbAPI
+        :return self: this movie, udpated with the relevant data from the given jsonObject
+        """
+        if jsonObject is not None:
+            for (key, value) in jsonObject.items():
+                if key in self.param_defaults.keys():
+                    if key == 'imdbRating' and not isinstance(value, float):
+                        try:
+                            value = float(value)
+                        except:
+                            value = None # TODO: we'll have to handle this upstream
```

Comment:
We might want to consider checking that imdbRating is not None before trying to display it in the ```results``` template.